rclpy/actions/minimal_action_client/examples_rclpy_minimal_action_client/client_asyncio.py

```diff
@@ -69,48 +70,49 @@ async def spinning(node):
 
 async def run(args, loop):
 
-    logger = rclpy.logging.get_logger('minimal_action_client')
-
     # init ros2
```

Comment:
:eyes:

```suggestion
    # init ROS 2
```

I know you didn't modify this, but just a suggestion :grin:

Reply:
Good call, done in 93877f8.